Formula/movgrab.rb

```diff
@@ -14,7 +14,11 @@ class Movgrab < Formula
     sha256 cellar: :any, high_sierra:   "7702b7817fc398f4901014bd6162578294da414a18b1ae4e5f10ef8cf05a678c"
   end
 
-  depends_on "libressl"
```

Comment:
I think we want to keep this. This was added in https://github.com/Homebrew/homebrew-core/pull/49738, and the refereced issue (https://github.com/ColumPaget/Movgrab/issues/3) is still open.

Good chance that this just disables SSL support entirely. We can probably just drop the `revision` bump too.

Reply:
I'm going to close this and align the Linux formula instead.